client/source/components/questionsAnswers/DisplayPhotos.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PhotoContainer } from './styles/Containers.styled';
+
+const DisplayPhotos = function CreateDisplayPhotosComponent() {
+
+  return (
+    <PhotoContainer>
+      <img
+        src="https://images.unsplash.com/photo-1477420143023-6a0e0b04b69a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
```

Comment:
I saw that you had test data. Good idea, so you can avoid hardcoding this here. Pass that test data in as if it were the real deal.